13 - JavaScript Modules and Using npm/es6modules-FINISHED/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "app.js",
   "scripts": {
+	// npm run build
```

Comment:
i don't think you can do this, please google

Reply:
this is a comment that I copied from the video.